app/models/manageiq/providers/inventory/persister/builder.rb

```diff
@@ -43,6 +43,9 @@ def self.prepare_data(name, persister_class, options = {})
         builder
       end
 
+      attr_accessor :name, :persister_class, :properties, :inventory_object_attributes,
+                    :default_values, :dependency_attributes, :options, :adv_settings, :shared_properties
```

Comment:
These can be useful for the provider persisters to access, e.g. https://github.com/ManageIQ/manageiq-providers-oracle_cloud/pull/10/files#diff-1f2902bb0b7601fd89db7a0dd19160cf103dfd3d86b1a313e66e56ab7a47a5a4L44